### PR TITLE
Fix CityBanners for Exp1

### PR DESCRIPTION
**Pull request by the-m4a**
**WHAT CHANGED**

- Fixed the CityBanners display for Rise and Fall.
- ~~CityBanners.xml from Exp1 now matches Exp2~~ Reverted this.  CityBannersInstances.xml is like the Exp1 unmodified version, plus the CQUI elements.
- Fixed minor issue in Exp2 version where if damage was exactly 40%, things would not display.  (Line 1161 in updated; all other changes are just tabs to spaces, or other space characters)
- Fixed up some formatting in both CityBanners.lua files in the Exp1 and Exp2 folders. (basically: so it'd be easier to make out the differences in the two files in the VSCode Diffing tool).

**TESTING**

- Tested in game.  Only mods enabled were Rise and Fall and CQUI.
- Verified City Banner shows districts, religion, loyalty.
- Verified Aerodome shows the airplane icon.
- Verified Encampment shows district strength and health bars

### CODE REVIEW

**Comment by the-m4a** [comment]
That's a good point, as there's no particular advantage.  My thought was largely based on feeling it would be unlikely that Firaxis would change anything with Exp1 at this point and it may be easier to maintain if the files matched.  I will make that adjustment and upload a 2nd set of changes into this PR.

File: Assets/Expansion1/CityBanners/citybannermanager.lua [hunk 1]
```diff
@@ -1063,7 +1072,7 @@ function CityBanner:UpdateColor()
   if (self.m_Type == BANNERTYPE_CITY_CENTER) then
     self.m_Instance.CityBannerFill:SetColor( backColor );
     self.m_Instance.CityBannerFillOver:SetColor( frontColor );
-    self.m_Instance.CityBannerFillOut:SetColor( brighterBackColor );
```
Comment (the-m4a) [inline Assets/Expansion1/CityBanners/citybannermanager.lua:1066]: brighterBackColor was not defined anywhere.  XP2 version used frontColor in this spot, so I matched that.

File: Assets/Expansion1/CityBanners/citybannermanager.lua [hunk 2]
```diff
@@ -1513,21 +1523,6 @@ function CityBanner:UpdateGovernor(pCity:table)
   end
 end
 
--- CQUI : TODO check if it's still needed
```
Comment (the-m4a) [inline Assets/Expansion1/CityBanners/citybannermanager.lua:1516]: No callers of this function existed; did not exist in the XP2 version

**Comment by the-m4a** [comment]
Updated the XML to be the Exp1 version plus CQUI enhancements.  Required a small change in the Lua (a button is no longer a button and is now just an icon)

**Comment by alimulhuda** [comment]
My comment about the old changes might end up being too overly cautious, but I was trying to treat this like I would a long term project at work. So it is more just trying to prevent potential headache for future maintainers. If I seem too overly cautious let me know, from time to time my work habits might leak through.

**Comment by alexeyOnGitHub** [comment]
merging. my mod code reviews are not very useful, as you probably already noticed guys :)
I can only do a very primitive sanity check. but I trust the two of you.

**Comment by the-m4a** [comment]
@alexeyOnGitHub Any review is a good review!  Keep asking questions!  Those are good things :)  I appreciate it!

**Pull request merged**